### PR TITLE
Fix missing code in HowTo: Tabs article

### PR DESCRIPTION
**Pull request by web-padawan**
Changes proposed in this pull request:

- Added missing parts of code to [HowTo: Tabs](https://web.dev/components-howto-tabs/) based on the [source on GitHub](https://github.com/GoogleChromeLabs/howto-components/blob/master/elements/howto-tabs/howto-tabs.js),
- Fixed one copy-paste leftover, added missing `_`, backticks and `<kbd>` tags,
- Updated the arrow functions style to align with the current Prettier defaults.

### CODE REVIEW

File: src/site/content/en/blog/components-howto-tabs/index.md [hunk 1]
```diff
@@ -121,6 +121,12 @@ Event handlers that are not attached to this element need to be bound if they ne
       this._onSlotChange = this._onSlotChange.bind(this);
 ```
 
+For progressive enhancement, the markup should alternate between tabs and panels. Elements that reorder their children tend to not work well with frameworks. Instead shadow DOM is used to reorder the elements by using slots.
+
+```js
+      this.attachShadow({ mode: 'open' });
+```
```
Comment (web-padawan) [inline src/site/content/en/blog/components-howto-tabs/index.md:128]: See the [source](https://github.com/GoogleChromeLabs/howto-components/blob/079d0fa34ff9038b26ea8883b1db5dd6b677d7ba/elements/howto-tabs/howto-tabs.js#L67-L71).

File: src/site/content/en/blog/components-howto-tabs/index.md [hunk 2]
```diff
@@ -207,6 +213,13 @@ Give each panel a `aria-labelledby` attribute that refers to the tab that contro
 
 The element checks if any of the tabs have been marked as selected. If not, the first tab is now selected.
 
+```js
+      const selectedTab =
+        tabs.find((tab) => tab.selected) || tabs[0];
+```
+
+Next, switch to the selected tab. `_selectTab()` takes care of marking all other tabs as deselected and hiding all other panels.
```
Comment (web-padawan) [inline src/site/content/en/blog/components-howto-tabs/index.md:221]: See the [source](https://github.com/GoogleChromeLabs/howto-components/blob/079d0fa34ff9038b26ea8883b1db5dd6b677d7ba/elements/howto-tabs/howto-tabs.js#L161-L165).

File: src/site/content/en/blog/components-howto-tabs/index.md [hunk 3]
```diff
@@ -278,12 +290,12 @@ Add `tabs.length` to make sure the index is a positive number and get the modulu
     }
 ```
 
-`_lastTab()` returns the last tab.
+`_nextTab()` gets the tab that comes after the currently selected one, wrapping around when reaching the last tab.
```
Comment (web-padawan) [inline src/site/content/en/blog/components-howto-tabs/index.md:293]: See the [correct description](https://github.com/GoogleChromeLabs/howto-components/blob/079d0fa34ff9038b26ea8883b1db5dd6b677d7ba/elements/howto-tabs/howto-tabs.js#L229-L232).

**Comment by web-padawan** [comment]
A gentle ping on this PR. There are just a few fixes in one article - please take a look if someone has time 🙏

**Review by AaronForinton — APPROVED** [review]

**Comment by AaronForinton** [comment]
Thanks for these contributions @web-padawan. I'll definitely have eyes open for HTML indentation on other pages

**Comment by web-padawan** [comment]
Thanks for taking a look! For some reason the status check is still red, do I need to rebase?

**Comment by AaronForinton** [comment]
That's got something to do with my permissions, but it'll stay work if you merge. Not sure why it shows up like that but every PR I touch does and they all merge fine

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**